File: week3/ageCheck.py
# Input the name
name = input("What is your name?")
# Input the age
# At first the result of input() is a string
# We use int() to cast the string to an integer
age = input("How old are you?")
age = int(age)

# Similar algorithm to pseudo-code
if age < 18:
    print("You can't order beer")
else:
    print("You can order anything you like")

File: week3/ageCheck.cpp
// C++ version for illustration
#include <iostream>

using namespace std;

int main()
{
    // Set the variables
    int age;
    string name;

    cout << "What is your name?" << endl;
    cin >> name;

    cout << "How old are you?" << endl;
    cin >> age;

    if(age < 18)
    {
        cout << "You may not order beer" << endl;
    }
    else
    {
        cout << "Order anything you like" << endl;
    }

    return 0;
}